app/common/lib/ledgerUtil.js

```diff
@@ -424,6 +447,8 @@ const getTwitchDuration = (state, data, mediaKey) => {
     return 0
   }
 
+  if (data.event === twitchEvents.START) debugger
```

Comment:
@NejcZdovc did you mean to leave `debugger` statement in?

Reply:
it's still WIP